queries.sql

```diff
@@ -125,3 +125,95 @@ SELECT o.full_name, count(a.name) as number_of_animals
     on a.owner_id = o.id 
     GROUP BY o.full_name;
 
+------------------------------------------------------------------------
+-- JOIN TABLES
+------------------------------------------------------------------------
+
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name AS "Name of Animal", vets.name AS "Vet", date_of_visit FROM visits v LEFT JOIN vets ON
+v.vet_id = vets.id LEFT JOIN animals a ON v.animal_id = a.id WHERE vets.name = 'William Tatcher' ORDER BY date_of_visit DESC LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT vets.name AS "Vet",COUNT(*) FROM visits v LEFT JOIN vets ON v.vet_id = vets.id 
+    LEFT JOIN animals a
+    ON v.animal_id = a.id 
+    WHERE vets.name = 'Stephanie Mendez' 
+    GROUP BY vets.name;
```

Comment:
- [ ] @great work on your query however your query result is wrong Stephanie Mendez visited 4 animals as per the data provided,
![image](https://user-images.githubusercontent.com/61408860/206853279-c38bee63-4f05-42b1-a1bf-669e6c3ab9c9.png)